src/status_im2/subs/chat/chats.cljs

```diff
@@ -424,3 +424,9 @@
  :<- [:chat/inputs-with-mentions]
  (fn [[chat-id cursor]]
    (get cursor chat-id)))
+
+(re-frame/reg-sub
+ :shared-element-id
```

Comment:
looks weird, could you elaborate ?

Reply:
you can use root `:shared-element-id` directly in the views

Reply:
> you can use root `:shared-element-id` directly in the views

Good to know, thanks